tests/test_cli.py

```diff
@@ -13,6 +13,7 @@
 # Standard library imports
 from collections import namedtuple
 from os.path import basename, exists
+from sys import executable
```

Comment:
```suggestion
import sys
```

I think it's easy to shadow `executable` by another variable of the same name, so it's better to call it from the `sys` module instead.